bin/install-dependencies.sh

```diff
@@ -1,10 +1,21 @@
 #! /bin/bash
 
-venv/bin/pip install -U $(curl -s https://raw.githubusercontent.com/ckan/ckan/2.7/requirement-setuptools.txt)
-venv/bin/pip install -U $(curl -s https://raw.githubusercontent.com/ckan/ckanext-harvest/c21c1627992cbcf0043b7b6873ff899f6f66767f/pip-requirements.txt)
-venv/bin/pip install -U $(curl -s https://raw.githubusercontent.com/ckan/ckanext-dcat/master/requirements.txt)
-venv/bin/pip install -U $(curl -s https://raw.githubusercontent.com/ckan/ckanext-spatial/master/pip-requirements.txt)
-venv/bin/pip install -U $(curl -s https://raw.githubusercontent.com/alphagov/ckanext-s3-resources/master/requirements.txt)
-venv/bin/pip install -Ue 'git+https://github.com/ckan/ckan.git@ckan-2.7.4#egg=ckan'
-venv/bin/pip install -r venv/src/ckan/requirements.txt
-venv/bin/pip install -r requirements.txt
+pip=${1-'/usr/bin/env pip'}
+
+$pip install -U $(curl -s https://raw.githubusercontent.com/ckan/ckanext-harvest/c21c1627992cbcf0043b7b6873ff899f6f66767f/pip-requirements.txt)
+$pip install -U 'git+https://github.com/ckan/ckanext-harvest.git@c21c1627992cbcf0043b7b6873ff899f6f66767f#egg=ckanext-harvest'
```

Comment:
aren't we using our own fork on `https://github.com/alphagov/ckanext-harvest.git` ?

Reply:
Not as of this change, no.  See https://github.com/alphagov/ckanext-datagovuk/pull/117/files#diff-b4ef698db8ca845e5845c4618278f29aL2